tests/cluster_wide_install/kustomization.yaml

```diff
@@ -9,46 +7,25 @@ patchesJson6902:
 - target:
     group: rbac.authorization.k8s.io
     version: v1
-    kind: Role
-    name: manager-role
+    kind: ClusterRoleBinding
+    name: cass-operator-manager-rolebinding
   patch: |-
```

Comment:
Are these patches still needed since you created `config/cluster/kustomization.yaml`?

Reply:
Yes, since changing the namespace is not working as one would imagine in Kustomize. It won't replace all the instances (such as these).